code/datums/status_effects/debuffs.dm

```diff
@@ -981,38 +981,44 @@
 
 
 //~~~LC13 General Debuffs~~~
-
+#define CARBON_HALFSPEED /datum/movespeed_modifier/qliphothoverload
 /datum/status_effect/qliphothoverload
 	id = "qliphoth intervention field"
 	duration = 10 SECONDS
 	alert_type = null
 	status_type = STATUS_EFFECT_REFRESH
 	var/statuseffectvisual
-	var/originalstamina = 0
+	var/speed_memory = 0
```

Comment:
You could obliterate this entire variable by using TemporarySpeedChange(4, 10 SECONDS) from hostile, as it more or less does the exact same thing. It also lowers the quantity of code in `on_remove` since it's done automatically.

Reply:
Ill swap it with the temp speed change.